### ISSUE TITLE
Passing asset_id to Coral.Talk.render() hangs

### ISSUE THREAD

**Issue by kaichristiansen** [coralproject/talk#2176]
I am having a similar issue to this older closed issue. I commented there and maybe should reopen that? Not sure of the best protocol. https://github.com/coralproject/talk/issues/1720

I am trying to pass an asset_id to the render method and it hangs.I have installed the plugin which I can see in the node_modules folder inside the talk docker container installation. When I test this locally, I get a "Graph QL: error: Internal Error" popup at the bottom of the viewport and then a spinning waiter in the blank comment div. My code looks like this:

  Coral.Talk.render(document.getElementById('coral_talk_stream'), {
    talk: '<talk url>',
    lazy: true,
    asset_id: '92ae9e52-1fdf-11e7-a7a3-001851cded07'
 });

I have tried it with and without asset_url. If I comment out asset_id it works fine. Is it possible the plugin is not working or in inactive?

**Comment by wyattjoh** [comment]
What does the logs referencing the internal error show?

**Comment by kaichristiansen** [comment]
I am not sure which logs you are referring to. I checked the docker logs for the talk container and nothing out of the ordinary. The error message appears in the browser in a black div at the bottom of the view point. Can you point me to a relevant log? Thanks.

**Comment by immber** [comment]
@kaichristiansen The `talk-plugin-custom-asset-id` plugin isn't included with the base distribution of Talk, but is available as a separate open source plugin. How have added the plugin to Talk? Are you using docker onbuild to include it as an external plugin?

**Comment by kaichristiansen** [comment]
Yes. I have included it. I am using the docker onbuild image and including it as a server plugin.

**Comment by kaichristiansen** [comment]
Ah. I think I found the problem. The doc for the plugin on git (https://github.com/coralproject/talk-plugin-custom-asset-id)  shows adding the plugin with a line like this:

{"@coralproject/talk-plugin-custom-asset-id": "^1.1.0"}

which works for installing the plugin but does not work at runtime.  I changed it to:

"talk-plugin-custom-asset-id",

and then installed the plugin directly using git clone and now it is no longer hanging. It moves on to reveal another error related to the docker container and https/ssl but I will pursue that separately.
